src/main/java/racing/Name.java

```diff
@@ -0,0 +1,18 @@
+package racing;
+
+public class Name {
+
+    public static final int MAX_NAME = 5;
+    private final String name;
```

Comment:
상수와 인스턴스 변수와의 구분을 위해 개행을 추가해보면 좋을 것 같습니다.
```suggestion
    public static final int MAX_NAME = 5;
    
    private final String name;
```